pkg/monitoring/manager/metrics.go

```diff
@@ -0,0 +1,107 @@
+package monitoring
```

Comment:
You'll need to run a `make check-license` which adds license headers to all new files